packages/components/src/components/popover-message-checkbox/popover-message-checkbox.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Checkbox from '../checkbox';
+
+type TPopoverMessageCheckbox = {
+    defaultChecked?: boolean;
+    message: string;
+    name?: string;
+    onChange: React.FormEventHandler<HTMLInputElement> &
```

Comment:
just for my information, why did you add the first type to `onChange`?

bcause I checked the `checkbox` component and there we have `(e: React.ChangeEvent<HTMLInputElement> | React.KeyboardEvent<HTMLSpanElement>) => void` as the type for `onChange`